RNA_pipeline/rna_strand.wdl

```diff
@@ -0,0 +1,96 @@
+version 1.0
+
+
+workflow RNAStrandWorkflow {
+    input {
+        File input_bam
+        File refseq = "gs://cds-vep-data/ucsc_ncbi_refseq_curated_hg38_20240303.bed"
+
+        String sample_id
+        String docker_image = "us-docker.pkg.dev/depmap-omics/public/strand:24q2"
+
+        Int boot_disk_size=60
```

Comment:
Runtime attrs can exist just at the task level usually and I don't see much reason to expose them in the workflow, especially since this gets unwieldy when you have lots of tasks. (We always have the option to override them during job submission if we have `meta { allowNestedInputs: true }` inside the task.)